pkg/tfbridge/diff.go

```diff
@@ -209,7 +209,7 @@ func makePropertyDiff(name, path string, v resource.PropertyValue, tfDiff shim.I
 					kind = pulumirpc.PropertyDiff_ADD
 				}
 			default:
-				if d.RequiresNew {
+				if d.RequiresNew || other.Kind == pulumirpc.PropertyDiff_DELETE_REPLACE {
```

Comment:
I wonder if we should expand this to include the other `replace` cases as well.

Reply:
I am having trouble thinking of test cases for the other `replace` cases but adding them doesn't break any existing tests. I'll defer to you if you think we should include those.